### DISCUSSION TITLE
SKR v1.4 Turbo Pin

### DISCUSSION THREAD

**Discussion by 2021927** [PixiePlacer/PixiePlacer#15] [Q&A]
Hello PixiePlacer ,
I can't find the Pin2.05 in my SKR v1.4 Turbo, what can I do, please.

**Comment by Guytou11** [comment]
Hi 2021927,
Have you found out in the meantime where Pin 2.05 is ?
I am having the same issue on the SKR v1.4 Turbo.

**Comment by ThibaudetTevai** [comment]
Same here

**Comment by Guytou11** [comment]
Hi,  From what I found is that pin 2.05 is in fact pin 2.5 (the BED output).  
There are 2 typo's on the pin assignment page of the Pixie wiki page:
- Down LED must be pin 0.10  (you can find this info in the Marklin fimware)
- Vacuum Pump is pin 2.5
Pin 2.05 does not exist.

**Comment by ThibaudetTevai** [comment]
Perfect :) Thanks a lot @Guytou11  !!!

**Comment by ThibaudetTevai** [comment]
And about the vacuum Sensing modules ? Which pin is connected to the modules ?
